ui/StatusQ/src/StatusQ/Components/LoadingComponent.qml

```diff
@@ -0,0 +1,39 @@
+import QtQuick 2.13
+
+import StatusQ.Core.Theme 0.1
+import QtGraphicalEffects 1.12
+import StatusQ.Animations 0.1
+
+Rectangle {
+    id: root
+    property bool loading
```

Comment:
could you please make this a `Controls` with `background` and `contentItem` properties set instead?